src/shell.js

```diff
@@ -80,15 +80,15 @@ if (ENVIRONMENT_IS_NODE) {
   var nodeFS;
   var nodePath;
 
-  Module['read'] = function read(filename, binary) {
```

Comment:
Ok, then let's rename it to avoid the warning. Maybe `shell_read`? (a little longer but I assume closure would minify it anyhow)